docs/book/src/design/overview.md

```diff
@@ -0,0 +1,21 @@
+# Overview
+
+The Nautilus Wallet system consists of the following high systems:
+
+1. An underlying [ledger platform](ledger-platform.md) and its asset(s)
+2. The [Wallet](wallet.md) itself:
+   - The [Wallet TEE](wallet-tee.md) (server) hosts and provides access to the Wallet SGX enclave
+   - The [Wallet Connector app](wallet-connector-app.md) (client) securely connects wallet owners with their wallets
+3. [Asset services](asset-services.md), which help manage and govern the asset:
+   - The [KYC service](kyc-service.md)
+   - User verification service
```

Comment:
- We might, at the end of the book, want to add a glossary of terms.  We could even link it here to draw attention to it early on.  The topic at hand is very dense when it comes to the jargon involved.
- The `kyc-service.md` link appears to be broken.  Forgotten commit or intentional?

Reply:
A glossary is definitely a good idea.

The KYC service page still needs to be created, yeah: good catch! I removed the link, for now.